roofit/roofitcore/src/RooVectorDataStore.cxx

```diff
@@ -1489,24 +1489,25 @@ RooVectorDataStore::RealFullVector* RooVectorDataStore::addRealFull(RooAbsReal*
 }
 
 
-/// Trigger a recomputatin of the cached weight sums. Meant for use by RooFit
+/// Trigger a recomputation of the cached weight sums. Meant for use by RooFit
 /// dataset converter functions such as the NumPy converter functions
 /// implemented as pythonizations. Not meant to be part of the public user
 /// interface, so the interface might change without warning.
+/// @{
```

Comment:
I don't remember what just using `\@` does, did you check?

I thought it's like this:
```
/// \name <groupName>
/// Description ...
/// @{

c++

/// @}

Reply:
Search [here](https://root.cern.ch/doc/master/classRooAbsArg.html) for "legacy" to see how this is rendered.